pkg/repo/badgerdb/repo.go

```diff
@@ -23,6 +23,9 @@ func WithDefaultDB() option {
 		path := viper.GetString("db.badgerdb.path")
 		badgerOpts := badger.DefaultOptions(path)
 
+		// NOTE: The default value for in-memory storage of ValueThreshold is 1 MB.
```

Comment:
Indeed, it is constrained by badger.maxValueThreshold https://github.com/dgraph-io/badger/blob/v3.2103.5/options.go#L210